frontend/src/pages/Login.tsx

```diff
@@ -1,14 +1,108 @@
+import { FirebaseError } from "firebase/app";
+import { GoogleAuthProvider, signInWithPopup } from "firebase/auth";
+import { useState } from "react";
 import { Helmet } from "react-helmet-async";
+import { useNavigate } from "react-router-dom";
+import styled from "styled-components";
 
+// import app from "../../firebase";
+
+import { createUser } from "@/api/users";
 import { Page } from "@/components";
+import { auth } from "@/firebase";
```

Comment:
Was able to bring back `firebase.ts` by using an absolute import. I think the relative import was what was tripping up the compiler. 

FYI `@` is an alias to `src`:

`frontend/tsconfig.json`
```json
...
"paths": {
  "@/*": ["src/*"]
},
```